torch_glow/tests/nodes/addmm_test.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from tests.utils import jitVsGlow
+
+def test_addmm():
```

Comment:
Shouldn't we just test linear here?

Reply:
here what we test is actaully addmm, if this is correct then linear is correct as well.

Reply:
Yeah but we don't support addmm directly, only the addmm instances that can be lowered to from linear right?

Reply:
yeah... so you think we can have a seperate for nn.linear?